tests/internal/test/service.go

```diff
@@ -118,20 +130,21 @@ func GetServices(t *testing.T, cfg *configs.Config) Services {
 		t.Fatal(err)
 	}
 
-	// make sure all requested proposal keys are created
-	keyCount, err := km.InitAdminProposalKeys(ctx)
-	if err != nil {
-		t.Fatal(err)
-	}
+	// // make sure all requested proposal keys are created
+	// keyCount, err := km.InitAdminProposalKeys(ctx)
```

Comment:
I'd remove the commented code, or add a `TODO` if it's waiting for updates to the `InitAdminProposalKeys` implementation